sechub-developertools/scripts/sechub-api.sh

```diff
@@ -459,9 +459,9 @@ function sechub_project_assign_user {
 
 
 function generate_sechub_project_create_data {
-  local PROJECT_ID="${1,,}"  # ,, converts to lowercase
+  local PROJECT_ID="$(tr [A-Z] [a-z] <<< "$1")"  # ,, converts to lowercase
```

Comment:
Would it be possible to use a more known pipe (`|`) here? Or is `<<<` a must-have?

Reply:
Ok, never mind, here string is probably fine. I just didn't see these used a lot in bash scripts.

Reply:
Btw, I took this from this comment: https://stackoverflow.com/a/2264435/12464594

Reply:
To be honest, I just copied this from StacOverflow ([link](https://stackoverflow.com/a/2264435/12464594)) 😅 But now I've read up on the difference between `|` and `<<<` ([link](https://unix.stackexchange.com/a/80372)) and it looks like `"$(echo "$1" | tr [A-Z] [a-z])"`  should also work, since `tr` doesn't set any variables or anything like that. I'm fine with either version 😊